webserver/templates/user/import.html

```diff
@@ -7,22 +7,34 @@ <h2 class="page-title">{{ user.musicbrainz_id }}</h2>
   {% if current_user.is_authenticated() and current_user == user %}
 
      <h3>Import from Last.fm</h3>
-     {% if loader %}
+     {% if loader_parse and loader_api %}
         <p>
           <strong>Step 2</strong>
         </p>
         <div class="well">
           <p>
              Drag this bookmarklet to your bookmarks toolbar:
           </p>
-          <p>
-             <span class="btn btn-warning btn-lg" style="width: 400px"><a href="{{loader}}" style="color: white;">Import Last.fm to ListenBrainz</a></span>
-          </p>
-        </div>
+					<a href="{{loader_parse}}" style="color: white;">
+						<p>
+							<span class="btn btn-warning btn-lg" style="width: 400px"> Bookmarklet </span>
+						</p>
+					</a>
         <p>
           <span style="color:red">IMPORTANT</span>: If you've saved this bookmarklet before 29 Sept, please delete the old
           bookmarklet and save it again, since we've moved to https.
         </p>
+					<p style="font-weight:bold;"> or </p>
```

Comment:
As i mentioned in the main comment thread -- we should remove the "scraping" option from the HTML pages -- it will only confuse people and it produces bad data.